projects/online/online/utils/dataloading.py

```diff
@@ -155,15 +155,18 @@ def data_iterator(
                 x = read_channel(fname, f"{channel}")
                 frames.append(x.value)
 
-                state_channel = f"{ifo}:GDS-CALIB_STATE_VECTOR"
+                if ifo in ["H1", "L1"]:
```

Comment:
I think we should make `state_channels` a configurable dict from `ifo` --> `channel_name`.  Will do this in a later PR